Assets/MirageXR/Tests/NewUI/ProfileView.cs

```diff
@@ -278,9 +216,9 @@ private void OnClickPrivacyPolicy()
 
     private void UpdatePrivacyPolicyButtonActive()
     {
-        var setActive = (DBManager.privacyPolicyDomain != string.Empty) ? true : false;
+        //var setActive = (DBManager.privacyPolicyDomain != string.Empty) ? true : false;
 
-        _btnPrivacyPolicy.gameObject.SetActive(setActive);
+        //_btnPrivacyPolicy.gameObject.SetActive(setActive);
     }
```

Comment:
Was it necessary to remove the "Open Privacy Policy" button?

Reply:
I disabled the button according to the Figma.  @fominykh Where is the Privacy Policy button located now?

Reply:
not sure what we planned... did we want to display it together with the server (because the policy itself can be different for each server)? @fwild

Reply:
Ok, we hide the "Open Privacy Policy" button when select custom server.